tetrad-lib/src/main/java/edu/cmu/tetrad/data/CovariancesFloatStandard.java

```diff
@@ -0,0 +1,160 @@
+
+package edu.cmu.tetrad.data;
+
+import static java.lang.Math.round;
+
+/**
+ * Compute covariance using the standard calculation.
+ * <p>
+ * Jan 27, 2016 4:37:44 PM
+ *
+ * @author Kevin V. Bui (kvb2@pitt.edu)
+ */
+public class CovariancesFloatStandard implements Covariances {
+    private final float[][] _data;
+    private final int numOfRows;
+    private final int numOfCols;
+    private float[][] covariances;
+
+    public CovariancesFloatStandard(double[][] data, boolean biasCorrected) {
+        _data = new float[data.length][data[0].length];
+
+        for (int i = 0; i < _data.length; i++) {
+            for (int j = 0; j < _data[0].length; j++) {
+                this._data[i][j] = (float) (round(data[i][j] * 100.0) / 100.0);
+            }
+        }
+
+        this.numOfRows = data.length;
+        this.numOfCols = data[0].length;
+        this.covariances = compute(biasCorrected);
```

Comment:
The compute method should be private if called inside of constructor.  If not, the user can call it again.